app/src/main/java/com/example/music_app/ui/screens/search/SearchScreen.kt

```diff
@@ -0,0 +1,104 @@
+package com.example.music_app.ui.screens.search
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyListState
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.example.music_app.R
+import com.example.music_app.data.models.Playlist
+import com.example.music_app.ui.screens.core.ButtonField
+import com.example.music_app.ui.screens.playlists.PlaylistsField
+import com.example.music_app.ui.theme.AppTheme
+
+@SuppressLint("FrequentlyChangedStateReadInComposition")
+@OptIn(ExperimentalMaterial3Api::class)
+@Preview
```

Comment:
Please don't add preview annotation directly to the composables - create separate functions with dummy data passed